pyramid_oereb/contrib/print_proxy/mapfish_print.py

```diff
@@ -170,25 +170,35 @@ def archive_pdf_file(pdf_archive_path, binary_content, extract_as_dict):
         return path_and_filename
 
     @staticmethod
-    def get_wms_url_params():
+    def get_wms_url_params(params=None):
         """
         Returns the list of additionally configured wms_url_params.
+        Args:
+            params: optional argument used for given custom parameters. Expecting a dict
```

Comment:
```suggestion
            params (dict): Optional argument used for given custom parameters.
```

See another function to have the right syntax to generate the documentation (I'm not sure for your "return" doc)